Search/scrape_plugins.py

```diff
@@ -71,8 +73,11 @@ def main():
     # Make sure fallback directories contain any skipped engines.
     verifyEngines(engines)
 
-    # Write the list of engine names for each locale.
-    writeList(engines)
```

Comment:
Is it possible for us to keep just one list and parse that to find the default engine for each locale?